invenio_app_rdm/records_ui/utils.py

```diff
@@ -55,3 +56,67 @@ def set_default_value(record_dict, value, path, default_prefix="metadata"):
         path = "{}.{}".format(default_prefix, path)
 
     dict_set(record_dict, path, value)
+
+
+def external_resources(record):
+    """Make number compact if too long."""
```

Comment:
```suggestion
def get_external_resources(record):
    """Generate external resource links for a record."""
```

nit: naming and docstring